metals/src/main/scala/scala/meta/internal/metals/WorkspaceSymbolProvider.scala

```diff
@@ -22,7 +21,7 @@ final class WorkspaceSymbolProvider(
     val workspace: AbsolutePath,
     statistics: StatisticsConfig,
     val buildTargets: BuildTargets,
-    val index: OnDemandSymbolIndex,
+    val index: scala.meta.internal.mtags.GlobalSymbolIndex,
```

Comment:
-> `GlobalSymbolIndex` ?

Reply:
Thanks, should be fixed now. I had underlooked that commit I think.